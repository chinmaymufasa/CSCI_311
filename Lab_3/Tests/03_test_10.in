2
160
5
2
78
5
2
66
5
2
167
5
2
196
5
-1
