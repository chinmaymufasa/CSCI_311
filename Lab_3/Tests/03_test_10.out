160 
78 160 
66 78 160 
66 78 160 167 
66 78 160 167 196 
